pkgs/development/python-modules/aiohomekit/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, cryptography
+, fetchFromGitHub
+, poetry
+, pytest-aiohttp
+, pytestCheckHook
+, zeroconf
+}:
+
+buildPythonPackage rec {
+  pname = "aiohomekit";
+  version = "0.2.60";
+
+  src = fetchFromGitHub {
+    owner = "Jc2k";
+    repo = pname;
+    rev = version;
+    sha256 = "03llk5i22hq163x568kz0qar5h0sda8f8cxbmgya6z2dcxv0a83p";
+  };
+
+  format = "pyproject";
+
```

Comment:
```suggestion
disabled = pythonAtLeast "3.9";

```

Reply:
@FRidh should we keep packages like this unconstrained and hope for upstream to fix them?

Reply:
At least from my point of view it would be better to constraint them because otherwise nixpkgs-review spits out broken packages which are known.

Reply:
Added, thanks.